packages/legacy/core/App/navigators/OnboardingStack.tsx

```diff
@@ -41,11 +42,14 @@ const OnboardingStack: React.FC = () => {
   const OnboardingTheme = theme.OnboardingTheme
   const carousel = createCarouselStyle(OnboardingTheme)
   const Onboarding = container.resolve(TOKENS.SCREEN_ONBOARDING)
-  const { pages, splash, useBiometry, preface } = useConfiguration()
+  const { pages, splash, useBiometry } = useConfiguration()
   const defaultStackOptions = createDefaultStackOptions(theme)
   const navigation = useNavigation<StackNavigationProp<AuthenticateStackParams>>()
   const onTutorialCompleted = container.resolve(TOKENS.FN_ONBOARDING_DONE)(dispatch, navigation)
   const terms = container.resolve(TOKENS.SCREEN_TERMS)
+  const developer = container.resolve(TOKENS.SCREEN_DEVELOPER)
+  const preface = container.resolve(TOKENS.SCREEN_PREFACE)
```

Comment:
Why preface from here and not `useConfiguration()`?

Reply:
Just slowly starting the process of making more use of the DI/IoC structure that we're going with. There's gonna be a bit of an awkward phase while we have configuration in two places but I thought we were moving away from overriding components from the configuration context

Reply:
Should `developer` and `preface` be capitalized since they are React component?